Src/MoneyFox/Converter/PaymentTypeIntConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace MoneyFox.Converter
+{
+    using Core.Aggregates.Payments;
+    using Core.Resources;
+    using System;
+    using System.Globalization;
+    using Xamarin.Forms;
+
+    public class PaymentTypeIntConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
Same here regarding unit test :)

Reply:
Thanks for the feedback. It all makes sense. I was conflicted on how best to approach the enum. My original thought was to use a nullable variable with the existing enum but the binding wasn’t happy with null values. I considered modifying the existing enum or adding a new one but didn’t love that either, so I took the easy route and just used casting with an int. But you’re right, it makes the code more difficult to understand. I’ll create a new enum as you suggested. This should all be fairly easy to implement so I should have a PR ready in a day or two.

Reply:
Sounds good!
Just to be sure: you don't have to create a new PR. Just push the changes to this branch and the PR is updated automatically.

Reply:
Maybe to add on this: it makes sense to not use an existing enum as nullable as well as not to extend an existing one. Especially since those have a different purpose (defining a payment type vs defining a filter).
A int is an easy an cheap solution. But just creating a new enum is also quite cheap and has in my opinion some benefits over an int :)

Reply:
Sounds good to me. I'll switch from int to a new enum and push updates to the PR. Thanks!